fission/src/ui/panels/configuring/assembly-config/interfaces/ConfigureGamepiecePickupInterface.tsx

```diff
@@ -117,16 +122,16 @@ const ConfigureGamepiecePickupInterface: React.FC<ConfigPickupProps> = ({ select
             return
         }
 
-        const gizmo = new TransformGizmos(
+        const gizmo = new GizmoSceneObject(
             new THREE.Mesh(
                 new THREE.SphereGeometry(0.5),
                 World.SceneRenderer.CreateToonMaterial(ReactRgbaColor_ThreeColor(theme.HighlightSelect.color))
-            )
+            ),
+            "translate",
+            1.5
         )
 
-        ;(gizmo.mesh.material as THREE.Material).depthTest = false
-        gizmo.AddMeshToScene()
-        gizmo.CreateGizmo("translate", 1.5)
+        ;((gizmo.obj as THREE.Mesh).material as THREE.Material).depthTest = false
```

Comment:
Same thing here. We could avoid the whole ESLint vs Prettier disagreement about lines starting with a semicolon as well as the double cast and generally ugly line by creating a function to set the `depthTest` field of the material. Also, is this something that needs to happen every time the gizmo is created? If so, it would be worth putting it inside of some function in the gizmo object so that we don't need to remember to call it every time.